test/realtime/UserThreadNonBlockedTest.cpp

```diff
@@ -112,6 +112,8 @@ class UserThreadNonBlockedTest : public ::testing::Test
             // Create subscriber
             subscriber_attr_.topic.topicDataType = type_.getName();
             subscriber_attr_.topic.topicName = "Dummy";
+            subscriber_attr_.times.initialAcknackDelay.seconds = 10;
+            subscriber_attr_.times.heartbeatResponseDelay.seconds = 10;
```

Comment:
What is the purpose of these changes? Was this test failing with the default values?

Reply:
Yes, because I was testing a mutex (locking it) that it also can locked by the heartbeat response event and initial ack event. With these value I'm trying to avoid that.

Reply:
:+1: